services/app/app/templates/portal/directory-sections.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="card mb-4">
+  <div class="card-body">
+    <p class="lead">Select your directory categories.</p>
+    <p>For each section below, select the directory categories your company should be featured in. You may select up to <code>{{ config.onlineDirectorySectionMax }}</code> categories per section.</p>
+    <p>{{ i18n "companyDetailsP3Label"}}</p>
+  </div>
+</div>
+
+<div class="row">
+  {{#each (get-edge-nodes this "model.websiteSites") as |site|}}
+    {{#if site.sections.edges.length}}
+      {{leadership/site site=site selectedIds=categories onUpdate=(action "update")}}
```

Comment:
Create new components for directory sections, don't re-use the leadership components. These can have independent rules -- for example the leadershipSectionMax is a limitation of the leaders program, and shouldn't be used with directory scheduling.